README.rst

```diff
@@ -245,6 +247,11 @@ MIT
 Change Log
 ----------
 
+22.1.27
```

Comment:
Generally we put unreleased here but I'll fix when ever I get around to releasing.
```suggestion
Unreleased
```